homeassistant/components/tuya/number.py

```diff
@@ -292,6 +292,52 @@
             device_class=NumberDeviceClass.TEMPERATURE,
         ),
     ),
+    # Tank Level Sensor
+    # Note: Undocumented
+    "ywcgq": (
+        NumberEntityDescription(
+            key=DPCode.MAX_SET,
+            translation_key="liquid_high_level",
+            entity_category=EntityCategory.CONFIG,
+            native_unit_of_measurement=PERCENTAGE,
+            icon="mdi:arrow-collapse-up",
+            native_min_value=0,
+            native_max_value=100,
+            native_step=1,
+        ),
+        NumberEntityDescription(
+            key=DPCode.MINI_SET,
+            translation_key="liquid_low_level",
+            entity_category=EntityCategory.CONFIG,
+            native_unit_of_measurement=PERCENTAGE,
+            icon="mdi:arrow-collapse-down",
+            native_min_value=0,
+            native_max_value=100,
+            native_step=1,
+        ),
+        NumberEntityDescription(
+            key=DPCode.INSTALLATION_HEIGHT,
+            translation_key="installation_height",
+            entity_category=EntityCategory.CONFIG,
+            device_class=NumberDeviceClass.DISTANCE,
+            native_unit_of_measurement=UnitOfLength.METERS,
+            icon="mdi:human-male-height",
+            native_min_value=100,
+            native_max_value=3000,
+            native_step=1,
```

Comment:
These are provided by the API (and the current provided ones in the code doesn't match the API specification as it seems).

```suggestion
```